File: CODE-IOS/SwiftPractice/Tech_OC_01/Tech_OC_01/Person.m
//
//  Person.m
//  Tech_OC_01
//
//  Created by bss on 11/28/15.
//  Copyright © 2015 DevilTaining. All rights reserved.
//

#import "Person.h"

@implementation Person

// 方法的实现
- (int)run:(int)numbers {
    
    printf("you should run %d steps\n",numbers);
    int j = 100;
    j = j + numbers;
    printf("j = %d\n",j);
    
    return j;
}

@end

File: CODE-IOS/SwiftPractice/SwiftAndOC_mix/SwiftAndOC_mix/Person.swift
//
//  Person.swift
//  SwiftAndOC_mix
//
//  Created by bss on 1/9/16.
//  Copyright © 2016 DevilTaining. All rights reserved.
//

import Foundation

class Person: NSObject {
    var name: String?
    init(name: String) {
        self.name = name;
    }
    deinit {
        print("the person named: \(self.name) say goodby")
    }
    
    func buyGoods() {
        let car:Card? = Card()
        car!.money = "253"
        car!.pay(23)
        Card.getCard(23242)
        
        
        
    }
}